### ISSUE TITLE
Add most new event metrics to VPN app

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5400]
Add event metrics in code, and confirm their behavior (by looking at logs). Non-event metrics are being done [https://mozilla-hub.atlassian.net/browse/VPN-4039](https://mozilla-hub.atlassian.net/browse/VPN-4039|smart-link).

**The canonical list is kept in** [*this spreadsheet*](https://docs.google.com/spreadsheets/d/1EdPD4bbSWW4VVfWPcQt4YqSB_Zpecfu1_f_5DKALAdw/edit#gid=0)**.** ﹍**The following list may be out of date**﹍**. This list also includes metrics that have both code and data dashboard components, as well as metrics that need extra data added to existing events:**

* # device-sessions
* # Views of the ‘Verification unsuccessful’ screen
* # Views of the ‘VPN tips and tricks’ onboarding screen
* # Clicks on the CTA of the ‘VPN tips and tricks’ onboarding screen
* # user-triggered “server location switched” events
* # of times a client reaches ‘No Signal'
* # of times a client reaches 'Unstable Connection'
* # of speed test is performed

* % of speed tests that resulted in n speed category

* # ‘Messages’ view opened
* # and % of devices that have opted-into / out-off AutoConnect on Startup
* # and % of device  who are using a non-system language
* # and % of sessions and subscribers that have activated ‘adblock DNS’
* # and % of sessions and subscribers that have activated ‘anti-tracking DNS’
* # and % of sessions and subscribers that have activated ‘custom DNS’
* # and % of sessions and subscribers that have activated at least 1 app exclusion
* # avg number of app exclusions for subscribers who’ve excluded at least 1
* # clicks / taps on ‘Contact Support’
* # support cases submitted
* # clicks / taps on View log
* # clicks/taps on ‘About Us’
* # clicks/taps on ‘Settings’ (the gear)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-3630)
┆Reporter: Matt Cleinman

**Comment by data-sync-user** [comment]
➤ Matt Cleinman commented:

I’ve unblocked this ticket on the additional Glean work, and instead created a new ticket for confirming iOS metrics (https://mozilla-hub.atlassian.net/browse/VPN-4037 ( https://mozilla-hub.atlassian.net/browse/VPN-4037|smart-link ) ) and have changed that Glean block over to that ticket.

This ticket, when complete, should mean all metrics in the spreadsheet (linked above) are complete on all non-IOS platforms, with the exceptions of metrics being done in https://mozilla-hub.atlassian.net/browse/VPN-3731 ( https://mozilla-hub.atlassian.net/browse/VPN-3731|smart-link ) and https://mozilla-hub.atlassian.net/browse/VPN-4038 ( https://mozilla-hub.atlassian.net/browse/VPN-4038|smart-link ), which are being broken out into their own tickets.
